### PR TITLE
[OLD] Add custom clipboard copy instead of relying on native select text

### PR DESCRIPTION
**Pull request by Fatxx**
**Description**

Improve UX by enabling one-click copy on the hex text field under the Transaction Review Data component.

**Proposed Solution**

https://recordit.co/QMjju83N9n

**Checklist**

* [x] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

**Screenshots/Recordings**

_If applicable, add screenshots or recordings to visualize the changes_

**Issue**
[CD3](https://github.com/MetaMask/mobile-planning/issues/307#issuecomment-1160848284)
Progresses https://github.com/MetaMask/mobile-planning/issues/307

### CODE REVIEW

**Review by gantunesr — APPROVED** [review]
LGTM

File: locales/languages/en.json [hunk 1]
```diff
@@ -582,7 +582,7 @@
     "mainnet": "Mainnet",
     "test_network_name": "Test networks",
     "custom_network_name": "Custom networks",
-		"popular": "Popular",
+    "popular": "Popular",
     "delete": "Delete",
```
[the remark this answers is not in the document]
Comment (Fatxx) [reply locales/languages/en.json:586]: Pff, IDK what's happening on .json files... I'll have a look

File: locales/languages/en.json [hunk 2]
```diff
@@ -1064,36 +1065,36 @@
...
-		"network_name": "{{networkName}} Network",
-		"network_added": " is now available in the network selector.",
-		"provider": "A provider is trusted to tell your wallet its balances and to broadcast its transactions faithfully",
-		"no_match": "No matching results found.",
-		"empty_popular_networks": "You’ve added all the popular networks. You can discover more networks",
-		"add_other_network_here": "here.",
-		"you_can": "Or you can",
-		"add_network": "add more networks manually."
+    "network_infomation": "This allows this network to be used within MetaMask",
+    "network_endorsement": "MetaMask does not endorse custom networks or their security.",
```
[the remark this answers is not in the document]
Comment (Fatxx) [reply locales/languages/en.json:1069]: I'm using spaces, was this file somehow generated externally or by some process? Might be the case

**Comment by Fatxx** [comment]
Please use https://github.com/MetaMask/metamask-mobile/pull/4579 in favour of this one that got merged by mistake.

**Pull request merged**